app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -288,7 +296,29 @@ protected void load() {
 
     private void loadUrl() {
         setWebSettings(true);
-        mWebView.reloadUrl(mItem.getUrl());
+        reloadUrl(mItem.getUrl());
+    }
+
+    private void reloadUrl(String url) {
+        reloadUrl(url, null);
+    }
+
+    private void reloadUrl(String url, File pdfFile) {
+        if (mPdfAndroidJavascriptBridge != null) {
+            mPdfAndroidJavascriptBridge.cleanUp();
+            mWebView.removeJavascriptInterface("PdfAndroidJavascriptBridge");
+        }
+        if (isPdfRenderingSupported() && url.equals(PDF_LOADER_URL)) {
```

Comment:
Let's replace `url.equals(PDF_LOADER_URL)` with `PDF_LOADER_URL.equalsIgnoreCase(url)` or `TextUtils.equals(url, PDF_LOADER_URL)` to avoid NPE if url is null?

Reply:
Okay, makes sense